bundles/mapping/mapmodule/AbstractMapModule.js

```diff
@@ -29,6 +29,7 @@ import './plugin/realtime/RealtimePlugin';
 import './plugin/mylocation/MyLocationPlugin';
 import './plugin/zoombar/Portti2Zoombar';
 import './plugin/panbuttons/PanButtons';
+import '../layerswipe/plugin/LayerSwipePlugin';
```

Comment:
We don't need to import it here as it is it's own bundle. We can just start the layerswipe bundle instead on the embedded map.